.buildkite/main-tests.json

```diff
@@ -2,7 +2,7 @@
   "tests": [
     {
       "test_name": "build-gnu",
-      "command": "RUSTFLAGS=\"-D warnings\" cargo build --release",
+      "command": "CROSVM_USE_SYSTEM_VIRGLRENDERER=1 CROSVM_USE_SYSTEM_MINIGBM=1 RUSTFLAGS=\"-D warnings\" cargo build --release",
```

Comment:
Instead of defining this every time, could we have a file somewhere in our tree where we define all the environment variables we need for testing and then do `source file.sh` here?

Reply:
make sense, I think I can just instead set the env_var at the top of this file and then it's applied.

Reply:
@dorindabassey yeah, that is even better if supported!

Reply:
I see, but using a file allow us to just change that file in the future if we need to remove/add new var envs.

Reply:
Yeah I think it's okay to leave the env vars as is in `"command"`.

Reply:
@dorindabassey okay I see, I thought it was more easy, sorry for the noise, it's fine.